CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contribution guide
+
+Welcome! We are glad that you want to contribute to our project! 💖
+
+This project accepts contributions via Github pull requests.
+
+This document outlines the process to help get your contribution accepted.
+
+There are many ways to contribute:
+
+* Suggest [features](https://github.com/bouvet/nord-juice-shop/issues/new?assignees=&labels=%3Abulb%3A+feature+request&template=feature-request.md&title=)
+* Suggest [changes](https://github.com/bouvet/nord-juice-shop/issues/new?assignees=&labels=%3Awrench%3A+change&template=change-request.md&title=)
+* Report [bugs](https://github.com/bouvet/nord-juice-shop/issues/new?assignees=&labels=%3Abug%3A+bug&template=bug-report.md&title=)
+
+You can start by looking through the [good first issues](https://github.com/bouvet/nord-juice-shop/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+## Fork the repository
+
+In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr).
+
+Here's a quick guide:
+
+1. Create your own fork of the repository
+2. Clone the project to your machine
+3. To keep track of the original repository, add another remote named upstream
+```shell
+git remote add upstream git@github.com:bouvet/nord-juice-shop.git
+```
+4. Create a branch locally with a succinct but descriptive name and prefixed with change type.
+```shell
+git checkout -b feature/my-new-feature
+```
+5. Make the changes in the created branch.
+6. Add the changed files
+```shell
+git add path/to/filename
+```
+7. Commit your changes using the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) formatting for the commit messages.
+```shell
+git commit -m "conventional commit formatted message"
+```
```

Comment:
Do we use this for anything? Maybe we should file an issue to add release please? Imo it's a hard requirement, as we could still squash merge with conventional commit message too, if it's important at the main branch. Should we really require conventional commits at local topic branches?

Reply:
We do not enforce it currently, but if so it should only apply to `main` (?). I imported most of these non-code (i.e. README's) files from another project of mine, so there may be some non-applicable items there.

Reply:
Issue created, see #51